appinfo/routes.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @author Hemant Mann <hemant.mann121@gmail.com>
+ *
+ * @copyright Copyright (c) 2017, ownCloud GmbH.
+ * @license GPL-2.0
+ * 
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ * 
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ */
+
+$app = new \OCA\Files_external_dropbox\AppInfo\Application();
+
+$app->registerRoutes($this, [
+	'routes' => [
```

Comment:
it is allowed to simply return the array - no need to call registerRoutes

Reply:
I wrote this based on the lines of documentation [https://doc.owncloud.org/server/latest/developer_manual/app/routes.html#routing](https://doc.owncloud.org/server/latest/developer_manual/app/routes.html#routing)

and also @PVince81 had done the same thing in [https://github.com/owncloud/files_external_gdrive/blob/master/appinfo/routes.php#L22](https://github.com/owncloud/files_external_gdrive/blob/master/appinfo/routes.php#L22)

So I thought this is the right standard

Reply:
I did not say your code is wrong. I just Wanted to point you to an easier way;-)

Reply:
Yeah returning an array is easier but I am writing full code because I find it more understandable rather than finding the magic which allows to return array :)